OWNERS

```diff
@@ -0,0 +1,12 @@
+# The OWNERS file is used by prow to automatically merge approved PRs.
+
+approvers:
+- mattmoor
+- n3wscott
+- vaikas-google
```

Comment:
remove vaikas-google? @vaikas

Reply:
I thought about it, but there were some repo's that still have `vaikas-google` so I opted to play it safe (knative/serving for example).  Happy to remove if requested.

Reply:
It's fine to remove, thanks for doing this!

Reply:
done in 36c9430